pkg/compiler/testdata/all.txt

```diff
@@ -14,6 +14,7 @@ foo$9(a ptr[out, ptr[in, string]])
 foo$10(a ptr[out, buffer[in]])
 foo$11(a int64[1:100, 2])
 foo$12(a int64[0:-1, 0x1000])
+foo$13(a int8[-255:255])
```

Comment:
Please add more tests, they are cheap. For begin/end separately, for bitfields.

Reply:
Ah, bitfields. Forgot about these. I'll add more tests for bitfields and will fix the bitsize in `IntType.CheckConsts`.